openapitools.json

```diff
@@ -1,7 +0,0 @@
-{
-  "$schema": "./node_modules/@openapitools/openapi-generator-cli/config.schema.json",
-  "spaces": 2,
-  "generator-cli": {
-    "version": "7.1.0"
-  }
-}
```

Comment:
Is it okay to remove this file? Are there any side effects?

Reply:
No side effects that I saw when I ran the generator